cinema/urls.py

```diff
@@ -1 +1,33 @@
-# write urls here
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from cinema.views import (GenreViewSet, ActorViewSet, CinemaHallViewSet,
+                          MovieViewSet, MovieSessionViewSet)
+
+router = DefaultRouter()
+router.register(
+    "genres",
+    GenreViewSet
+)
+router.register(
+    "actors",
+    ActorViewSet
+)
+router.register(
+    "cinema_halls",
+    CinemaHallViewSet
+)
```

Comment:
```suggestion
router.register("genres", GenreViewSet)
router.register("actors", ActorViewSet)
router.register("cinema_halls", CinemaHallViewSet)
```
If you can write the code in one line, it’s better to do so.

Reply:
fixed